lighthouse-core/computed/metrics/lantern-metric.js

```diff
@@ -85,7 +94,7 @@ class LanternMetricArtifact {
   /**
    * @param {LH.Artifacts.MetricComputationDataInput} data
    * @param {LH.Audit.Context} context
-   * @param {any=} extras
+   * @param {Omit<Extras, 'optimistic'>=} extras
```

Comment:
since requirements aren't changing very quickly any more, at some point we should take a pass at these `extras` to see if we can untangle them a bit :)